### ISSUE TITLE
Sub Parts do not disable mirroring

### ISSUE THREAD

**Issue by DrprofLuigi** [DonBruce64/MinecraftTransportSimulator#649]
```
{
	"definitions" : [//Dyes: Black=0 Red =1 Green=2 Brown=3 Blue=4 Purple=5 UNU=6 Silver=7 Grey=8 Pink=9 Lime=10 Yellow=11 Light Blue=12 Magenta=13 Orange=14 White=15
		{
			"name" : "Steve",
			"subName" : "",
			"extraMaterials" : []
		},
		{
			"name" : "Derrick_Amithson",
			"subName" : "_derrick_amithson",
			"extraMaterials" : []
		},
		{
			"name" : "Dr_prof_Luigi",
			"subName" : "_dr_prof_luigi",
			"extraMaterials" : []
		},
		{
			"name" : "ยง6EndrTheStrange (Patron)",
			"subName" : "_endrthestrange",
			"extraMaterials" : []
		}
	],

	"general": {
		"type" : "custom",
		"customType" : "npc",
		"disableMirroring" : true,
		//"materials" : ["minecraft:leather:0:8", "minecraft:wool:0:2"] //Vanilla
		"materials" : ["minecraft:leather:0:8", "minecraft:wool:0:2"] //Immersive Engineering
	},
	
	"custom" : {
		"width" : 0.5,
		"height" : 1
	}
}
```

Bumper Stickers work fine, and they use the same `disableMirroring`

![2020-12-09_01 02 52](https://user-images.githubusercontent.com/36775575/101608363-77b54200-39ba-11eb-8e61-36311c99bfca.png)

**Comment by DonBruce64** [comment]
subParts inherit the mirroring of their parents.  The idea being that if the parent is mirrored, the sub-parts should be as well.  Is this an incorrect assumption?  Because for things like gun pods and wheel hubcaps you want the mirroring to occur.

**Comment by DonBruce64** [comment]
Fixed as of V19.12.0.
